samcli/local/apigw/authorizers/lambda_authorizer.py

```diff
@@ -24,10 +24,26 @@ def __init__(self, identity_source: str):
         """
         self.identity_source = identity_source
 
-    @abstractmethod
     def is_valid(self, **kwargs) -> bool:
         """
         Validates if the identity source is present
+
+        Parameters
+        ----------
+        kwargs: dict
+            Key word arguments to search in
+
+        Returns
+        -------
+        bool:
+            True if the identity source is present
+        """
+        return self.find_identity_value(**kwargs) is not None
+
+    @abstractmethod
+    def find_identity_value(self, **kwargs) -> Any:
```

Comment:
why Any?

Reply:
Was initially done since in theory, these values could be anything, however in practice they are suppose to be string values, or missing. Will update the typing in this class